libpod/options.go

```diff
@@ -21,6 +15,11 @@ import (
 	"github.com/opencontainers/runtime-tools/generate"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"net"
```

Comment:
These are in the wrong location  We like thr golang default imports to be at top of import and github imports at bottom with a blank line separating them.